packages/pilot/src/containers/TransactionDetails/index.js

```diff
@@ -642,7 +643,9 @@ class TransactionDetails extends Component {
       city: formattedCustomer && formattedCustomer.city,
       complementary: formattedCustomer && formattedCustomer.complementary,
       document_number: formattedCustomer && formattedCustomer.document_number,
-      email: formattedCustomer && formattedCustomer.email,
+      email: formattedCustomer && formattedCustomer.email
```

Comment:
aqui não precisa do ternario, apenas retornar o truncate onde antes retornava o e-mail é suficiente
```suggestion
      email: formattedCustomer && (<Truncate text={formattedCustomer.email} />)
```

Reply:
resolvido!